crowdpact/apps/pact/models.py

```diff
@@ -25,15 +30,33 @@ class Pact(models.Model):
     goal = models.IntegerField()
     deadline = models.DateTimeField()
 
+    # State variables for handling the endgame
+    goal_met = models.BooleanField(default=False)
+    notification_events_created = models.BooleanField(default=False)
+
     objects = PactManager()
 
     @property
+    def has_enough_pledges(self):
+        """
+        Does this Pact have enough pledges?
+        """
+        return self.pledge_count >= self.goal
+
+    @property
     def pledge_count(self):
         """
         Return the number of pledges for this Pact.
         """
         return self.pledge_set.count()
 
+    @property
+    def is_ongoing(self):
+        """
+        Return if the pledge is still ongoing (ie, we've not hit the deadline).
+        """
+        return utc_tz.localize(datetime.utcnow()) < self.deadline
```

Comment:
Django's storing datetimes in a datetime-aware fashion.  It's an option in the settings.py.  Not sure if it'll cause more problems than it solves, but I thought I'd leave it enabled to see.